homeassistant/components/lyric/sensor.py

```diff
@@ -116,6 +132,30 @@ class LyricSensorEntityDescription(
     ),
 ]
 
+ACCESSORY_SENSORS: list[LyricSensorAccessoryEntityDescription] = [
+    LyricSensorAccessoryEntityDescription(
+        key="room_temperature",
+        translation_key="room_temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda _, accessory: accessory.temperature,
+        suitable_fn=lambda _, accessory: accessory.temperature
+        if accessory.type == "IndoorAirSensor"
+        else False,
```

Comment:
```suggestion
        suitable_fn=lambda _, accessory: accessory.type == "IndoorAirSensor",
```

Reply:
Unless there is a reason why `accessory.temperature` isn't Trueish in some cases. This also prevents a bug where the sensor isn't added if the accessory temperature is exactly 0 (since translating that to a bool makes it false)

Reply:
Nice catch!  This has been fixed in my most recent commit.